src/main/java/com/zendesk/maxwell/MaxwellReplicator.java

```diff
@@ -352,6 +385,14 @@ public RowMap getRow() throws Exception {
 						setReplicatorPosition((AbstractBinlogEventV4) v4Event);
 					}
 					break;
+				case MySQLConstants.ROTATE_EVENT:
```

Comment:
is there a use case to stop on log file rotation? my understanding was that mysql would send eof when catching up to master (no more events to process) https://github.com/zendesk/open-replicator/pull/22, not necessarily on log rotation